content/20.results.md

```diff
@@ -25,34 +25,39 @@ Although Bioinformatics was established in 1998 and BMC Bioinformatics in 2000,
 Therefore, without first and middle names, we do not have author gender predictions before this year.
 
 There is a slow increase of the proportion of predicted female authors, arriving at just over 20% on average in 2019 (Fig. {@fig:gender_breakdown}A).
-We observe very similar trend within each journal, but estimated female proportion has increased the least in _PLOS Computational Biology_ (Supplementary Fig S1).
+We observe very similar trend within each journal, but estimated female proportion has increased the least in _PLOS Computational Biology_ (Supplementary Fig. S1).
 ISCB fellows and keynote speakers appear to be more evenly split between men and women compared to the population of authors published in computational biology and bioinformatics journals (Fig. {@fig:gender_breakdown}B); however, it has not yet reached parity.
 The overall increasing female proportion trend seems to be present in each honor category, especially in the group of ISCB Fellows (Fig. {@fig:gender_breakdown}C), which seems to have markedly increased after 2015.
 PSB has only one or two keynotes per year, so its estimated proportion of female speakers varies considerably (Fig. {@fig:gender_breakdown}C).
 However, the 2020 PSB keynotes were the only set of honorees across our analysis for which nearly all the probability was ascribed to female speakers.
 
 
-![Estimated proportion of gender prediction over the years of all Pubmed journal authors (A),
-  of all ISCB fellows and keynote speakers (B),
-  and of ISCB honorees in each honor category (C).
+![Estimated composition of gender prediction over the years of 
```

Comment:
When I read papers I really like it if the first line of a figure legend is the take-home point of the figure. If the reader takes away only one thing from this figure, what should it be?

Reply:
That's a great point! I made adjustments, let me know what you think!